docs/releases/versioning-policy.md

```diff
@@ -1 +1,71 @@
+active support for bug fixes. The features in the latter two phases are both experimental.
+The features in alpha phase are mostly stable. On a case-by-case basis, they will either
+stay in the maintenance mode or be improved to general availability in the future. The features
+in beta phase are complete but haven't been tested extensively. They are subject to
+future deprecation.
+
+## Core features in general availability
+
+### APIs
+
+|                  | Upload | Create | Get | List | Delete | Archive | Unarchive | Enable | Disable | Terminate | Retry |
+|:----------------:|:------:|:------:|:---:|:----:|:------:|:-------:|:---------:|:------:|:-------:|:---------:|:-----:|
+| Pipeline         | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+| Pipeline Version | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+| Run              | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |        |         | - [x]     | - [x] |
+| Job              | - [x]  |  - [x] |- [x]|- [x] | - [x]  |         |           | - [x]  | - [x]   |
+| Experiment       | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+
+* Refer to the Kubeflow Pipelines [introduction]((https://www.kubeflow.org/docs/pipelines/overview/)) for
+  - An [overview](https://www.kubeflow.org/docs/pipelines/overview/pipelines-overview/) and [main concepts](https://www.kubeflow.org/docs/pipelines/overview/concepts/), for example, what is a pipeline or run in Kubeflow Pipelines.
+  - How to manage the Kubeflow Pipelines resources via the the [Kubeflow Pipelines UI](https://www.kubeflow.org/docs/pipelines/overview/interfaces/).
+* Refer to the Kubeflow Pipelines [SDK samples and tutorials](https://www.kubeflow.org/docs/pipelines/tutorials/sdk-examples/)
+for how to manage the Kubeflow Pipelines resources via the Kubeflow Pipelines client.
+* A complete API method list and a detailed description of parameters to those API methods are available at the Kubeflow Pipelines [API reference](https://www.kubeflow.org/docs/pipelines/reference/api/kubeflow-pipeline-api-spec/).
+
+### SDK
+
+* Only [ComponentSpec](https://github.com/kubeflow/pipelines/blob/master/sdk/python/kfp/components/structures/components.json_schema.json) in SDK is in general availability phase.
+
+## Experimental features
+
+### Features in Alpha
+
+* Job API
+
+|                  | Upload | Create | Get | List | Delete | Archive | Unarchive | Enable | Disable | Terminate | Retry |
+|:----------------:|:------:|:------:|:---:|:----:|:------:|:-------:|:---------:|:------:|:-------:|:---------:|:-----:|
+| Job              | - [x]  |  - [x] |- [x]|- [x] | - [x]  |         |           | - [x]  | - [x]   |
+
+
+* SDK DSL
+
+* [Upgrade/Reinstall the Kubeflow Pipelines instance](https://www.kubeflow.org/docs/pipelines/upgrade/).
+
+
+## Features in Beta
+
+* SDK client (that is, the helper functions). The helper functions are to be
```

Comment:
rather than to be deprecated, we may mention they are maintained by community, may not be feature complete and lacks automated testing. Some of the helper methods are still relatively easier to use than auto-generated ones, so we still see value in those.

If we can drive the community improve its status, I'm ok with graduating it into beta later.

Reply:
change to "The helper functions mainly relies on community maintenance", which implies little support from google side.